package.yaml

```diff
@@ -136,3 +136,7 @@ tests:
     - fmt
     - morley-ledgers
     - data-default
+    - hedgehog
+    - tasty-hedgehog
+    - o-clock
+    - containers
```

Comment:
nitpick, but can you keep this list sorted please?

Reply:
Yes.